src/content/quickstart/gke/_index.md

```diff
@@ -0,0 +1,163 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "Google GKE"
+weight: 20
+---
+
+Google Kubernetes Engine is a platform for containerized applications provided by google,
+you can find more about the platform [here](https://cloud.google.com/kubernetes-engine)
+
+
+## Setup the command line utility
+
+Download the google cloud sdk: 
+
+> https://cloud.google.com/sdk
+
+Find more detailed instructions for your platform here:
+
+> https://cloud.google.com/sdk/docs/quickstarts
+
+Make sure the necessary components of the gcloud shell are installed
+
+```
+source ~/.bashrc # if you just installed the gcloud shell
+gcloud components install kubectl beta
+gcloud init
+```
+
+
+## Create and deploy cluster A
+
+In this step you will deploy cluster A, with the default IP CIDRs
+
+| Pod CIDR        | Service CIDR |
+|-----------------|--------------|
+|192.168.128.0/18 |172.17.0.0/16 |	
+
+
+```bash
+gcloud container clusters create cluster-a \
+            --machine-type=g1-small \
+            --cluster-ipv4-cidr=192.168.128.0/18 \
+            --services-ipv4-cidr=172.17.0.0/16 \
+            --enable-ip-alias 
+    
+```
+
+The create cluster step will take some time, you can create Cluster B in parallel if you wish.
+
+{{% notice info %}}
+  
+**Notes about network CIDRs**: Please note that the default VPC subnets per zone
+in google cloud (10.128.0.0/9) for worker nodes can conflict with the Cluster CIDRs
+for some k8s distributions, see: https://cloud.google.com/vpc/docs/vpc#ip-ranges
+{{% /notice %}}
+
+
+## Create and deploy cluster B
+
+In this step you will deploy cluster B, modifying the default IP CIDRs
+
+| Pod CIDR        | Service CIDR   |
+|-----------------|----------------|
+|192.168.192.0/18 |10.4.128.0/19   |
+
+```bash
+gcloud container clusters create cluster-b \
+    --machine-type=g1-small \
+    --cluster-ipv4-cidr=192.168.192.0/18 \
+    --services-ipv4-cidr=10.4.128.0/19 \
+    --enable-ip-alias 
+
+```
+
+## Make your clusters ready for submariner
+
+Submariner gateway nodes need to be able to accept traffic over ports 4500/UDP and 500/UDP
+when using IPSEC. In addition we use port 4800/UDP to encapsulate traffic from the worker nodes
+to the gateway nodes and ensuring that Pod IP addresses are preserved.
+
+```bash
+gcloud compute firewall-rules create submariner-traffic \
```

Comment:
for me this just opened udp:500, so the last one. I had to generate 3 separate rules, one for each port

Reply:
add INGRESS and EGRESS rules